### ISSUE TITLE
Deployment Fails to Start

### ISSUE THREAD

**Issue by lightiv** [ovrclk/akash#1278]
### Issue type

Bug

### Akash version

v0.12.1

### Akash token balance

Enter keyring passphrase:
- name: skynet-tehp9
  type: local
  address: 
  pubkey: 
  mnemonic: ""
  threshold: 0
  pubkeys: []
---
balances:
- amount: "1044511489"
  denom: uakt
pagination:
  next_key: null
  total: "0"

### What happened

I deploy my node and it errors out with:

[osmosis-6dcb65db4d-nd9d7] /bin/sh: 1: start.sh: not found

There is no reference to start.sh in my deploy.yml or my Dockerfile.  I did use start.sh in a prior image but dropped because of the same error.    This is driving me crazy as I spent the day working this out.  It finally worked and then just hours later it is back to the error above.

### What did you expect to happen

The app to deploy and execut per the deploy file.  If I log into the image locally and run the start command it work perfectly fine.

### How to reproduce it

Deploy the app.  See deploy.yml below

### Environment

mainnet-2 on Ubuntu 20.04

### Anything else we need to know?

---
version: "2.0"

services:
  osmosis:
    image: 
    command:
      - osmosisd start
    expose:
      - port: 8080
        as: 80
        to:
          - global: true
      - port: 26657
        to:
          - global: true
      - port: 26656
        to:
          - global: true
      - port: 1317
        to:
          - global: true
      - port: 9090
        to:
          - global: true

profiles:
  compute:
    osmosis:
      resources:
        cpu:
          units: 1
        memory:
          size: 2Gi
        storage:
          size: 32Gi
  placement:
    dcloud:
      attributes:
        host: akash
      signedBy:
        anyOf:
          - akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63
      pricing:
        osmosis:
          denom: uakt
          amount: 100

deployment:
  osmosis:
    dcloud:
      profile: osmosis
      count: 1
---
leases:
- escrow_payment:
    account_id:
      scope: deployment
      xid: /1333302
    balance:
      amount: "0"
      denom: uakt
    owner: akash10cl5rm0cqnpj45knzakpa4cnvn5amzwp4lhcal
    payment_id: 1/1/akash10cl5rm0cqnpj45knzakpa4cnvn5amzwp4lhcal
    rate:
      amount: "6"
      denom: uakt
    state: open
    withdrawn:
      amount: "0"
      denom: uakt
  lease:
    created_at: "1333327"
    lease_id:
      dseq: "1333302"
      gseq: 1
      oseq: 1
      owner: 
      provider: akash10cl5rm0cqnpj45knzakpa4cnvn5amzwp4lhcal
    price:
      amount: "6"
      denom: uakt
    state: active
pagination:
  next_key: null
  total: "0"
---

## Next steps

You may [join developer chat](https://akash.network/chat) for community support.

**Comment by troian** [comment]
can you post content of the Dockerfile you've used to build to image

**Comment by lightiv** [comment]
I am going to close this.  It looks like I need to always specify the full path to the deploy file.  I still do not know how it could be looking to a different folder but apparently it was.  When I added the full path to the deployment file the problem went away.
```
akash provider send-manifest <FULL_PATH_TO>/deploy.yml --node $AKASH_NODE --dseq $AKASH_DSEQ --provider $AKASH_PROVIDER --home ~/.akash --from $AKASH_KEY_NAME
```
<FULL_PATH_TO> = ~/folder/deploy.yml

I have been able to deploy three in a row with issue.
